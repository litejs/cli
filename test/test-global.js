
describe(function() {
	for (var i = 1015; i--; ) this.should("break stack", function(assert) {assert.ok(1).end()})
	this
	.should("have globals defined", function(assert) {
		assert.type(global.describe, "function")
		assert.type(global.test, "function")
		assert.type(global.it, "function")
		assert.type(global.should, "function")
		assert.end()
	})
})


