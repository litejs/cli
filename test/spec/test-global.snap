# pass  1016/1016 [1019/1019]
